feature-group-definitions/canvas-text.yml

```
@@ -0,0 +1,11 @@
+name: Canvas text metrics
+description: The `fillText()` and  `strokeText()` methods draw text to 2D canvas. The `measureText()` method measures the width and other metrics which can be used to position the text.
```

nit (could also be "to 2D canvas**es**")

```suggestion
description: The `fillText()` and  `strokeText()` methods draw text to a 2D canvas. The `measureText()` method measures the width and other metrics which can be used to position the text.
```